pkg/controller/argocd/custommapper.go

```diff
@@ -96,3 +97,30 @@ func (r *ReconcileArgoCD) tlsSecretMapper(o handler.MapObject) []reconcile.Reque
 
 	return result
 }
+
+func (r *ReconcileArgoCD) namespaceResourceMapper(o handler.MapObject) []reconcile.Request {
```

Comment:
Can you please put some comments on this function, describing what it does and why?

Reply:
added the comments! Thanks